src/main/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtil.java

```diff
@@ -446,26 +446,32 @@ public static void writeSubjectsElement(XMLStreamWriter xmlw, DatasetVersionDTO
                         for (HashSet<FieldDTO> fieldDTOs : fieldDTO.getMultipleCompound()) {
                             String subject = null;
                             String subjectScheme = null;
-                            String schemeURI = null;
+                            String keywordTermURI = null;
+                            String keywordVocabURI = null;
 
                             for (Iterator<FieldDTO> iterator = fieldDTOs.iterator(); iterator.hasNext();) {
                                 FieldDTO next = iterator.next();
                                 if (DatasetFieldConstant.keywordValue.equals(next.getTypeName())) {
                                     subject = next.getSinglePrimitive();
                                 }
 
+                                if (DatasetFieldConstant.keywordTermURI.equals(next.getTypeName())) {
+                                    keywordTermURI = next.getSinglePrimitive();
+                                }  
+
                                 if (DatasetFieldConstant.keywordVocab.equals(next.getTypeName())) {
                                     subjectScheme = next.getSinglePrimitive();
                                 }
-
+                                
                                 if (DatasetFieldConstant.keywordVocabURI.equals(next.getTypeName())) {
-                                    schemeURI = next.getSinglePrimitive();
+                                    keywordVocabURI = next.getSinglePrimitive();
                                 }
                             }
 
                             if (StringUtils.isNotBlank(subject)) {
                                 subject_check = writeOpenTag(xmlw, "subjects", subject_check);
-                                writeSubjectElement(xmlw, subjectScheme, schemeURI, subject, language);
+                                // we prioritize the keywordTermURI metadata to populate schemeURI
+                                writeSubjectElement(xmlw, subjectScheme, StringUtils.isNotBlank(keywordTermURI) ? keywordTermURI : keywordVocabURI, subject, language);
```

Comment:
@stevenferey  - The DataCite Schema includes both a schemeURI and valueURI attributes - should this now include both rather than swapping the term URI into the schemeURI attribute?

Reply:
@qqmyers, The branch is updated with the modification of the OpenAire export. Thanks a lot